src/components/Miscellaneous/Badge/Badge.vue

```diff
@@ -42,14 +62,25 @@ export default {
   computed: {
     style() {
       const backgroundColorIsHexColor = isHexColor(this.backgroundColor);
-      
+
       return `
         background: ${backgroundColorIsHexColor ? this.backgroundColor : `var(--color-${this.backgroundColor})`} !important;
         color: var(--color-${this.color}) !important;
-        display: ${this.wrapContent ? 'inline-block' : ''};
+        display: ${this.wrapContent ? 'inline-flex' : ''};
```

Comment:
i think this can break some interfaces, can't it?

Reply:
I'm not sure if it can break something, but with wrapContent true and 'display: inline-block' it looks like this
![image](https://user-images.githubusercontent.com/51389447/168645597-e8c80116-267a-45a3-a9dd-e2269454324c.png)
Instead of this
![image](https://user-images.githubusercontent.com/51389447/168646913-453841e4-1cdf-4a88-9506-d236893ef266.png)